src/commands/deleteContainerApp/IDeleteContainerAppWizardContext.ts

```diff
@@ -0,0 +1,11 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ExecuteActivityContext, IActionContext } from "@microsoft/vscode-azext-utils";
+import { ContainerAppTreeItem } from "../../tree/ContainerAppTreeItem";
+
+export interface IDeleteContainerAppWizardContext extends IActionContext, ExecuteActivityContext {
+    containerApp: ContainerAppTreeItem;
```

Comment:
Adding only the needed properties to the context rather than the entire tree item would be best. In this case I think it's `resourceGroupName`, `name`, and `subscription`. It's not a huge deal, but I think it's best not to put tree items in the context.